cocos/core/gfx/webgl/webgl-gpu-objects.ts

```diff
@@ -170,6 +170,7 @@ export interface IWebGLGPUFramebuffer {
     isOffscreen: boolean;
     width: number;
     height: number;
+    lodLevel: number;
```

Comment:
I don't think this is necessary since webgl doesn't allow mip levels other than 0